misc/specs/atrium-specs-common/src/main/kotlin/ch/tutteli/atrium/specs/integration/ResultFeatureAssertionsSpec.kt

```diff
@@ -81,8 +90,9 @@ abstract class ResultFeatureAssertionsSpec(
         }
```

Comment:
can you please add two cases to this context. One where everything succeeds and one where something is wrong (e.g. the Exception type or the message)

Reply:
What description would you recommend for the case that fails?

Reply:
Something like for the failure case of isSuccess on line 65